services/distribution/src/main/java/app/coronawarn/server/services/distribution/statistics/file/RemoteStatisticJsonFileLoader.java

```diff
@@ -0,0 +1,31 @@
+package app.coronawarn.server.services.distribution.statistics.file;
+
+import app.coronawarn.server.services.distribution.config.DistributionServiceConfig;
+import app.coronawarn.server.services.distribution.objectstore.client.ObjectStoreClient;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+@Component
+@Profile("!local-json-stats")
+public class RemoteStatisticJsonFileLoader implements JsonFileLoader {
+
+  @Autowired
+  @Qualifier("stats-s3")
+  ObjectStoreClient s3Stats;
```

Comment:
fields should  be private and injected via constructor for consistency with the rest of the code